src/codegen/generate-classes.ts

```diff
@@ -1868,11 +1868,50 @@ function generateZig(
     ...proto,
   };
 
-  const externs = Object.entries({
+  const gc_fields = Object.entries({
     ...proto,
     ...Object.fromEntries((values || []).map(a => [a, { internal: true }])),
-  })
-    .filter(([name, { cache, internal }]) => (cache && typeof cache !== "string") || internal)
+  }).filter(([name, { cache, internal }]) => (cache && typeof cache !== "string") || internal);
+
+  const cached_values_string =
+    gc_fields.length > 0
+      ? `
+    pub const gc = enum (u8) {
```

Comment:
what about `pub const props` or `pub const properties`?

although `gc` is nice because these are write barriers, not really properties